docs/standard/modelling/identifiers.rst

```diff
@@ -1,142 +1,94 @@
 
-  Most of the identifier types listed below **are not** suitable for publication as part of an open dataset.
+  Most of the identifier types listed below are not suitable for publication as part of an open dataset.
 
+The following identification types can currently be used in BODS. Suggestions for new types should be made through the `issue tracker <https://github.com/openownership/data-standard/issues>`_.
 
-The following identification types are currently documented. Suggestions for new types should be made through the `issue tracker <https://github.com/openownership/data-standard/issues>`_. 
+**PASSPORT**
 
-PASSPORT
-++++++++
-
-Passport numbers should follow the format of the identifier (second) line in a machine-readable passport (see `Appendix B to Part 4 of ICAO Doc 9303 <http://www.icao.int/publications/Documents/9303_p4_cons_en.pdf>`_) including at least the document number. 
+Passport numbers SHOULD follow the format of the identifier line in a machine-readable passport (see `Appendix B to Part 4 of ICAO Doc 9303 <http://www.icao.int/publications/Documents/9303_p4_cons_en.pdf>`_) including at least the document number.
 
 Parsers should be able to extract the document number from the first 9 characters, and to access any subsequent information supplied according to the ICAO format.
 
-TAXID
-+++++
-
-Country taxpayer identification systems vary. Where specific guidance on including numbers from a particular jurisdiction is required, this may be included here in future.
-
-IDCARD
-++++++
+**TAXID**
 
-Country ID card systems vary. Where specific guidance on including numbers from a particular jurisdiction is required, this may be included here in future.
+Taxpayer identification regimes vary from country to country. Where guidance on including numbers from a particular jurisdiction is required, this may be included here in future.
 
+**IDCARD**
 
-Multiple Identifiers for entities or people
--------------------------------------------
+ID card systems vary. Where guidance on including numbers from a particular jurisdiction is required, this may be included here in future.
 
-A source system might hold the following identifying information for a single company:
+Special case: internal identifiers
++++++++++++++++++++++++++++++++++++
 
-- A registered company number; and
-- A VAT number;
+Data management systems may generate internal identifiers for people, or for records about people, or both.
 
-In this case, two entries can be created in the Entity statement's ``identifiers`` array:
+If multiple records in the system are known to relate to the same person, the data management system SHOULD assign a unique identifier to the person, and publish this identifier in the associated records. The identifier SHOULD be included in the ``identifiers`` array with a ``schemeName`` value ‘{publisher name}-{identifier type}’. For example, 'AtlantisCorporateRegister-PersonReference'.
```

Comment:
```suggestion
If multiple records in the system can relate to the same person, the data management system SHOULD assign a unique identifier to the person, and publish this identifier in the associated records. The identifier SHOULD be included in the ``identifiers`` array with a ``schemeName`` value ‘{publisher name}-{identifier type}’. For example, 'AtlantisCorporateRegister-PersonReference'.
```

Reply:
this is a system wide thing right? not something that would be applied only to specific people with multiple records?

Reply:
There's quite a big difference between what we'd be asking with these two formulations. In yours: we're saying that if the system is designed in this way then they should go off and resolve people's identities and then issue unique per-person identifiers. In mine: we're saying that if the system is designed in this way AND they already resolve people's identities then they should issue unique per-person identifiers. 

So I'm going to leave it as-is, with the easier-to-meet requirement.